getting-started/faqs.md

```diff
@@ -0,0 +1,101 @@
+# Frequently Asked Questions
+
+## Business Users
+
+Design questions to influence purchasing decision
+
+### About CFT
+
+1. What is Cloud File Transfer?
+**Cloud File Transfer \(CFT\)** is a centralized, light-weight, fully managed, API-driven service that allows agencies or businesses to transfer files easily, securely and efficiently at scale.
+
+
+2. What are the benefits of using CFT?
+
+- Plug and play
```

Comment:
Can we rephrase the top 3 points? i crafted these to make it very sales-y. but FAQs should be more explanatory.